Tests/Functional/Hooks/TranslateHookTest.php

```diff
@@ -125,18 +130,20 @@ public function contentNotTranslateWithDeeplWhenLanguageNotSupported(): void
             ->withAttribute('normalizedParams', NormalizedParams::createFromServerParams($serverParams));
 
         /** @var TranslateHook $translateHook */
-        $translateHook = GeneralUtility::makeInstance(TranslateHook::class);
+        $translateHook = $this->get(TranslateHook::class);
+        $languageService = $this->get(LanguageService::class);
+        /** @var SiteFinder $siteFinder */
+        $siteFinder = $this->get(SiteFinder::class);
+        $siteConfig = $siteFinder->getSiteByPageId(1);
 
-        $languageService = GeneralUtility::makeInstance(LanguageService::class);
-        $siteConfig = $languageService->getCurrentSite('pages', 1);
-        $sourceLanguageRecord = $languageService->getSourceLanguage($siteConfig['site']);
+        $sourceLanguageRecord = $languageService->getSourceLanguage($siteConfig);
         $content = $translateHook->translateContent(
             'Hello I would like to be translated',
             $sourceLanguageRecord['language_isocode'],
             'BS'
         );
 
-        static::assertSame('Hello I would like to be translated', $content);
+        static::assertSame('', $content);
```

Comment:
Are you sure, this is correct? From getting a translated content to an empty string?